Src/Support/Google.Apis.Auth/OAuth2/ITokenAccess.cs

```diff
@@ -40,6 +38,6 @@ public interface ITokenAccess
         /// <param name="authUri">The URI the returned token will grant access to.</param>
         /// <param name="cancellationToken">The cancellation token.</param>
         /// <returns>The access token.</returns>
-        Task<string> GetAccessTokenForRequestAsync(string authUri = null, CancellationToken cancellationToken = default(CancellationToken));
```

Comment:
Possibly revert this file? While the change is fine in itself, it's not really doing anything.